wrapper.go

```diff
@@ -324,6 +332,16 @@ func (r *resticWrapper) runCommand(command string) error {
 					rCommand.scanOutput = shell.ScanBackupPlain
 				}
 			}
+
+			// Redirect a stream source to stdin of restic if configured
+			if source, err := r.prepareStreamSource(); err == nil {
+				if source != nil {
+					streamSource = source
+					rCommand.stdin = streamSource
+				}
+			} else {
+				return newCommandError(rCommand, "", fmt.Errorf("%s on profile '%s': %w", r.command, r.profile.Name, err))
+			}
 		}
```

Comment:
I noticed when running with --dry-run there's a race condition where the main command sometimes finishes before displaying the stdin command(s)

Reply:
Will have a look

Reply:
The sync leads to a better implementation as `restic` is only started after the command returned initial bytes. My attempts to fix this pass all local tests but fail in CI. Will check it later.

Reply:
After lots of fiddling, I finally found that `/usr/bin/sh -c "child-proc ..."` on Ubuntu has issues to propagate SIGINT signals to the child-proc (Ubuntu uses "dash" for "sh"). On unix "bash" is now used if available (with fallback to "sh") and it fixes the problem.

On MacOS "sh" maps to "bash" therefore the tests had succeeded there.